resources/views/inputs/tile-selection-option.blade.php

```diff
@@ -4,14 +4,17 @@
     'wireModel',
     'single' => false,
     'mobileHidden' => false,
+    'withIcon' => false,
+    'iconWrapper' =>  'flex flex-col justify-center items-center md:space-y-2 h-full',
```

Comment:
It's worth adding this to `tile-selection.blade.php` too, possibly with the same default. You don't actually need them when using `@include` as attributes pass down, but it's good to be explicit i think. @ItsANameToo do you agree? 🤔

Reply:
I think there's 2 ways of going about this - remove all the props from this file and only have them in `tile-selection.blade.php`, or make this a separate component and use with `<x-ark-tile-selection-option` and pass the properties in that way instead

Reply:
if this component will be used as standalone component it makes sense to have the props explicitly defined. If it's always part of the `tile-selection.blade.php` one then we can remove the props here if they are defined in the parent with defaults

Reply:
👍 it was originally intended to only be used with the parent. Can't think of a use right now of it being standalone. I guess for now, remove the props from the `option` component. To make it more obvious it's not intended to be used as a component it could be moved to the includes folder (`inputs/includes`)